cachito/web/models.py

```diff
@@ -62,6 +62,29 @@
 )
 
 
+class ContentManifest:
```

Comment:
Could you move this to a new file? `models.py` is already rather large and this change is not really related to a database schema.